converter_app/readers/stoe_raw.py

```diff
@@ -0,0 +1,204 @@
+import logging
+
+from converter_app.readers.helper.reader import Readers
+from converter_app.readers.ascii import AsciiReader
+
+logger = logging.getLogger(__name__)
+
+import struct
+import numpy as np
+
+
+
+class StoeRawReader(AsciiReader):
+    """
+    Implementation of the TVB Reader for reading Stoe raw binary data file.
+    Code was extracted from the TVB jupiter Notebook by Ron Dockhorn.
+    # Beginn of Notebook
+    # Simple Conversion of Stoe raw data file
+    # by Ron Dockhorn (https://orcid.org/0000-0002-5268-5430)
+    """
+    identifier = 'stoe_raw_reader'
+    priority = 5
+
+    contentrawfile = ''
+    file_type = ''
```

Comment:
Never use static variables in reader. This could lead to unexpected behavior. Use:
```python
    def __init__(*args, **kwargs):
        super().__init__(*args, **kwargs)
        self.contentrawfile = ''
        self.file_type = ''
```